droid-configs.inc

```diff
@@ -318,6 +318,8 @@ do_not_move_to_lib64=(
     "systemd/system"
     "systemd/user"
     "tmpfiles.d"
+    "sysctl.d"
+    "sysusers.d"
```

Comment:
Sorted:
```suggestion
    "sysctl.d"
    "systemd/system"
    "systemd/user"
    "sysusers.d"
    "tmpfiles.d"
```
P.S. This list is only for devices that are changing their adaptations from 32 to 64bit, but it's good to keep it updated as reference manual still.